### PR TITLE
added active class

### PR DESCRIPTION
**Pull request by biswajit035**
## Describe your changes
I have added a **ACTIVE** class so that user can get idea about his active tab in **NAVBAR**

## Screenshots - If Any (Optional)
AFTER

## Issue ticket number and link -
#138 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.

- [x] Followed the repository's [Contributing Guidelines](https://github.com/dvstechlabs/Noteslify/blob/main/CONTRIBUTING.md).

- [x] I ran the app and tested it locally to verify that it works as expected.

- [x] I have starred the repository.

### CODE REVIEW

**Comment by devarshishimpi** [comment]
@biswajit035 Kindly complete all steps as requested before review

File: frontend/src/components/Sidenav/Sidenav.css [hunk 1]
```diff
@@ -168,6 +168,10 @@ header .image-text .profession {
     color: var(--sidebar-color);
 }
 
+.active{
+    background-color: #e4e9f7;
+}
+
```
Comment (biswajit035) [inline frontend/src/components/Sidenav/Sidenav.css:174]: for active class background color

File: frontend/src/components/Sidenav/Sidenav.js [hunk 2]
```diff
@@ -89,7 +90,7 @@ const Sidenav = () => {
                       </a>
                   </li>
 
-                  <li className="nav-link">
+                        <li className={`nav-link ${location === '/bin' ? "active" : ""}`}>
```
Comment (biswajit035) [inline frontend/src/components/Sidenav/Sidenav.js:93]: For automatically adding the active class according to the url

File: frontend/src/components/Sidenav/Sidenav.js [hunk 3]
```diff
@@ -68,14 +69,14 @@ const Sidenav = () => {
...
-                  <li className="nav-link">
+                  <li className={`nav-link ${location === '/notes'?"active":""}`}>
                         <Link to={'/notes'}>
                           <i className="fa-solid fa-house icon"></i>
                           <span className="text nav-text">Dashboard</span>
                         </Link>
                   </li>
 
-                  <li className="nav-link">
+                        <li className={`nav-link ${location === '/myaccount' ? "active" : ""}`}>
```
Comment (biswajit035) [inline frontend/src/components/Sidenav/Sidenav.js:79]: For automatically adding the active class according to the url

File: frontend/src/components/Sidenav/Sidenav.js [hunk 4]
```diff
@@ -68,14 +69,14 @@ const Sidenav = () => {
               </li>
 
               <ul className="menu-links">
-                  <li className="nav-link">
+                  <li className={`nav-link ${location === '/notes'?"active":""}`}>
```
Comment (biswajit035) [inline frontend/src/components/Sidenav/Sidenav.js:72]: For automatically adding the active class according to the url

**Review by biswajit035 — COMMENTED** [review]
I have reviewed the changes

**Comment by biswajit035** [comment]
This is my 2nd PR of my first OPENSOURCE, So I have no idea what to do now. 
Can you please guide ?

**Review by abhinandanwadwa — APPROVED** [review]

**Comment by devarshishimpi** [comment]
> This is my 2nd PR of my first OPENSOURCE, So I have no idea what to do now. Can you please guide ?

That's it, it will be merged

**Review by devarshishimpi — APPROVED** [review]
LGTM!! Thanks For Contributing!!!

**Comment by biswajit035** [comment]
> That's it, it will be merged

It is merged but when I am going to https://noteslify.stonecss.com/notes this features is not working. 
but I pulled the code and run localy it is working fine.
Can you please help to make this features live?

**Comment by devarshishimpi** [comment]
Yes, you are right. The feature will be brought into production on the next release ( Maybe today or tomorrow )

**Pull request merged**